src/web/client/common/authenticationProvider.ts

```diff
@@ -25,7 +25,7 @@ export async function dataverseAuthentication(dataverseOrgURL: string): Promise<
         const session = await vscode.authentication.getSession(PROVIDER_ID, [`${dataverseOrgURL}//.default`, 'offline_access'], { createIfNone: true });
         accessToken = session.accessToken;
     } catch (error) {
-        vscode.window.showErrorMessage(ERRORS.AUTHORIZATION_FAILED);
+        vscode.window.showErrorMessage(localize("microsoft-powerapps-portals.webExtension.init", "Authorization Failed. Please run again to authorize it"));
```

Comment:
Please check if key is correct 'microsoft-powerapps-portals.webExtension.init'?

Reply:
Added unique keys